src/Microsoft.AspNetCore.Authentication.JwtBearer/Events/JwtBearerChallengeContext.cs

```diff
@@ -40,5 +42,15 @@ public JwtBearerChallengeContext(HttpContext context, AuthenticationScheme schem
         /// WWW-Authenticate header. This property is always null unless explicitly set.
         /// </summary>
         public string ErrorUri { get; set; }
+
+        /// <summary>
+        /// If true, will skip any default logic for this event.
+        /// </summary>
+        public bool Skipped { get; private set; }
```

Comment:
Using `EventResultState` had one merit... getting rid of all these bool properties.